etc/profile-m-z/youtube-dl-gui.profile

```diff
@@ -0,0 +1,58 @@
+# Firejail profile for youtube-dl-gui
+# Description: A cross platform front-end GUI of the popular youtube-dl media downloader
+include youtube-dl-gui.local
+# This file is overwritten after every install/update
+include globals.local
+
+#These are blacklisted by disable-interpreters.inc
+include allow-python2.inc
+include allow-python3.inc
+
+noblacklist ${HOME}/.config/youtube-dlg
+noblacklist ${VIDEOS}
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.config/youtube-dlg
+whitelist ${HOME}/.config/youtube-dlg
+whitelist ${VIDEOS}
+include whitelist-common.inc
+include whitelist-runuser-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+machine-id
+netfilter
+no3d
```

Comment:
This maybe decreases performances of ffmpeg while converting.

Reply:
I haven't noticed any performance degradation. Perhaps, on big files performance degradation can be noticeable.I will report you back on this.

Reply:
yep. it degrades performance.

Reply:
IMHO we should remove it then.

Reply:
> IMHO we should remove it then.

or rather keep it commented out. which one you prefer?

Reply:
TBH I don't care. It's not that extra protections that it's worth to be used if you only download slow files IMHO.

Reply:
download speed remains same,it's the post-download processing which is affected.I would like to keep it commented out.